ale_linters/graphql/gqlint.vim

```diff
@@ -1,9 +1,18 @@
 " Author: Michiel Westerbeek <happylinks@gmail.com>
 " Description: Linter for GraphQL Schemas
 
+function! ale_linters#graphql#gqlint#GetCommand(buffer) abort
+    let l:filename = expand('#' . a:buffer . ':t')
+
+    return ale#path#BufferCdString(a:buffer)
+    \   . 'gqlint'
+    \   . ' --reporter=simple ' 
```

Comment:
Linter says there's a space on the end of this line. Please remove, and this should be good to merge.